__tests__/app.test.js

```diff
@@ -45,7 +45,8 @@ describe("GET/api/", () => {
       it('responds with status 404', () => {
           return request(app)
           .get('/not-a-route')
-          .expect(404)
+          .expect(404).send({ msg: 'Route not found' });
+
```

Comment:
we don't want a `.send` here, you want to be testing what comes back from your error handler so you can test the message on the body of the error 
For example
```
test('status:400, responds with an error message when passed a bad user ID', () => {
  return request(app)
    .get('/api/users/notAnID')
    .expect(400)
    .then(({ body }) => {
      expect(body.msg).toBe('Invalid input');
    });
});
// ...
```
(taken from https://notes.northcoders.com/courses/js-back-end/error-handling)